### PR TITLE
New readme

### PR DESCRIPTION
**Pull request by Renthal**
New version of the README matching the new API-key management

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,209 +1,113 @@
...
+Once installed, `darwin` is accessible as a command line tool.
+A useful way to navigate the CLI usage is trough the help command `-h/--help` which will 
+provide additional information for each command available. 
 
-### Command line
 
-`darwin` is also accessible as a command line tool.
+### Client Authentication 
 
+To access perform remote operation on Darwin you first need to authenticate.
```
Comment (andreaazzini) [inline README.md]: I think this line can be improved a little
```suggestion
To perform remote operations on Darwin you first need to authenticate.
```

File: README.md [hunk 2]
```diff
@@ -1,209 +1,113 @@
...
+This requires a [team-specific API-key](https://darwin.v7labs.com/?settings=api-keys).  
+If you do not already have a Darwin account, you can [register for free](https://darwin.v7labs.com).
```
Comment (andreaazzini) [inline README.md]: You can't anymore register for free, let's get rid of this line, or replace it by something like
```suggestion
If you do not already have a Darwin account, you can [contact us](https://darwin.v7labs.com/whatever-link) and we can set one up for you.
```

File: README.md [hunk 3]
```diff
@@ -1,209 +1,113 @@
...
+NAME                 IMAGES     PROGRESS
+myteam/mydataset     112025        73.0%
 ```
 
 
-#### Create/remove a dataset 
+### Create/remove a dataset 
 
-Creates an empty dataset remotely.
+To creates an empty dataset remotely:
```
Comment (andreaazzini) [inline README.md]:
```suggestion
To create an empty dataset remotely:
```

File: README.md [hunk 4]
```diff
@@ -212,37 +116,37 @@ Supported extensions:
...
-| `remove`        | `project_name`           | str                | True      |
-|                 | `-r` `--remote`          | str                | True      |
-| `url`           | `project_name`           | str                |           |
-| `push`          | `project_name`           | str                | True      |
-|                 | `files`                  | str                | True      |
-|                 | `-e`, `--exclude`        | str                |           |
-|                 | `--fps`                  | int                |           |
\ No newline at end of file
+$ darwin dataset pull test 
+TODO update output
```
Comment (andreaazzini) [inline README.md]: Just in case, reminder!

Comment (Renthal) [reply README.md]: done!

File: README.md [hunk 5]
```diff
@@ -1,209 +1,113 @@
...
 ```
 $ darwin authenticate
-Username (email address): simon@v7labs.com
-Password: *******
-Project directory [~/.darwin/projects]: 
-Projects directory created /Users/simon/.darwin/projects
-Authentication succeeded.
 ```
 
+You will be then prompted to insert the API-key, whether you want to set the corresponding team as
```
Comment (simedw) [inline README.md]:
```suggestion
You will be then prompted to enter your API-key, whether you want to set the corresponding team as 
```

File: README.md [hunk 6]
```diff
@@ -1,209 +1,113 @@
...
-$ darwin create example-dataset
-Dataset 'example-project' has been created.
-Access at https://darwin.v7labs.com/datasets/example-project
+$ darwin dataset create test
+Dataset 'test' (myteam/test) has been created.
+Access at https://darwin.v7labs.com/datasets/579
 ``` 
 
-To delete the project on the server add the `-r` /`--remote` flag
+Note that in this case `579` will be the dataset ID.
```
Comment (simedw) [inline README.md]: I think we shouldn't really talk about dataset ids

File: README.md [hunk 7]
```diff
@@ -212,37 +116,33 @@ Supported extensions:
...
+[darwin_demo.py](https://github.com/v7labs/darwin-py/blob/new_README/darwin_demo.py).
+
+```python
+from darwin.client import Client
+from darwin.dataset.identifier import DatasetIdentifier
+
+client = Client.local(team_slug="myteam") 
+dataset_identifier = DatasetIdentifier.from_slug(dataset_slug="test", team_slug="myteam")
+ds = client.get_remote_dataset(dataset_identifier=dataset_identifier)
+ds.pull()
```
Comment (simedw) [inline README.md]:
```python
client = Client.local() # use the configuration in ~/.darwin/config.yaml
dataset = client.get_remote_dataset(DatasetIdentifier.parse("myteam/test"))
dataset.pull() # downloads annotations and images for the latest exported version
```
I feel like that is nicer minimal introduction

File: darwin/dataset/__init__.py [hunk 8]
```diff
@@ -1 +0,0 @@
-from darwin.dataset.remote_dataset import RemoteDataset
```
Comment (simedw) [inline darwin/dataset/__init__.py]: Let's not remove this import.

File: README.md [hunk 9]
```diff
@@ -1,209 +1,113 @@
...
-
-## Usage
-
-Darwin can be used as a python library or as a command line tool.
-Main functions are:
+Darwin-py can be used as a python library or as a command line tool (CLI).
+Main functions are (but not limited to):
 
 - Client authentication
 - Listing local and remote datasets
```
[the remark this answers is not in the document]
Comment (Renthal) [reply README.md]: Its the darwin-py repository, I would say its quite self explanatory

File: README.md (hunk 6 above)
Comment (Renthal) [reply README.md]: removed

File: README.md (hunk 7 above)
Comment (Renthal) [reply README.md]: adopted

File: darwin/dataset/__init__.py (hunk 8 above)
Comment (Renthal) [reply darwin/dataset/__init__.py]: agree

**Pull request merged**